scaaml/capture/aes/crypto_alg.py

```diff
@@ -30,7 +35,7 @@ class SCryptoAlgorithm(AbstractSCryptoAlgorithm):
 
     def __init__(self,
                  firmware_sha256: str,
-                 crypto_implementation,
+                 crypto_implementation: type[AESSBOX],
```

Comment:
Out of curiosity why is `type` subscript allowed here by pylint 3.8 but not the other place?

Reply:
Fixed it everywhere... just in case it decides to fail :)